doc/source/whatsnew/v1.2.1.rst

```diff
@@ -10,6 +10,18 @@ including other versions of pandas.
 
 .. ---------------------------------------------------------------------------
 
+Enhancements
```

Comment:
yeah can you make this api breaking instead (also add the ```.. whatsnew_121.api_breaking``` tag above this section